db/inst/import-try/01.global.subset.R

```diff
@@ -0,0 +1,36 @@
+# 1. Global subset
+library(data.table)
+library(bit64)
+message("Loading TRY data...")
```

Comment:
The goal was to have a bit more control, like many other logger packages used in other languages. In this case there are 5 different levels: severe, error, warn, info and debug. Each of these will print the message with the file it was generated in. In the case of severe it will also stop. You can set the level of the logger and have more or less output, for example setting it to warn, will only display warn, error and severe messages, but will not print the info and debug messages.

This is loosly based on the python logging module and the apache log4X modules.